storybook/stories/data/Hierarchy.ts

```diff
@@ -0,0 +1,88 @@
+export interface SunburstData {
+  name: string;
+  value?: number;
+  children?: SunburstData[];
+  fill?: string;
+}
+
+const hierarchy: SunburstData = {
```

Comment:
The file is called hierarchy, the type used here is SunburstData though. Should this exist as a separate file?

Reply:
I think this file would make a lot of sense if we were to use a shared type between TreeMap and SunburstChart data? Is that possible / does it even make sense? 

If not, I would recommend to integrate this data into the SunburstChart story.

Reply:
I think it would make sense to use the same type for both. But I think the `size` property should be re-named `value` since it could represent any quantitative value, and it should support a custom fill color. 

For now, I'll just delete this file and integrate the data into the sunburst story to avoid changing treemap.